orchestra/proc-macro/src/parse/parse_orchestra_struct.rs

```diff
@@ -300,18 +311,18 @@ impl Parse for Consumes {
 }
 
 #[derive(Debug, Clone)]
-pub(crate) struct MessageCapacity {
+pub(crate) struct ChannelCapacity<T: Parse> {
 	#[allow(dead_code)]
-	tag: kw::message_capacity,
+	tag: T,
 	#[allow(dead_code)]
-	colon_token: Token![:],
+	colon_token: token::Colon,
```

Comment:
nit: I think `Token![:]` is preferred

Reply:
It won't compile due to this [issue](https://github.com/rust-lang/rust/issues/50676).

Reply:
Thank you!